src/components/Card.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { IoIosAddCircle } from "react-icons/io";
+import { useNavigate } from "react-router-dom";
+
+const Card = () => {
+  const [data, setData] = useState([]);
+  const navigate = useNavigate();
+  function handleDelete(email) {
+    const items = JSON.parse(localStorage.getItem("phoneBook"));
+    const index = items.indexOf(email);
+    items.splice(index, 1);
+    setData(items);
+    localStorage.setItem("phoneBook", JSON.stringify(items));
+  }
+  useEffect(() => {
+    const items = JSON.parse(localStorage.getItem("phoneBook"));
+    setData(items);
+  }, []);
+  return (
+    <div className="flex justify-center items-center">
+      <section>
+        <div className="py-16">
+          <div className=" px-6 max-w-6xl text-gray-500">
+            <div className="flex items-center justify-center">
+              <div className="text-center">
+                <h2 className="text-3xl text-gray-950 dark:text-white font-semibold">
+                  Quick call your Friends
+                </h2>
+                <p className="mt-6 text-gray-700 dark:text-gray-300">
+                  A Phone book Application with some more Information.
+                </p>
+              </div>
+              <div>
+                <IoIosAddCircle
+                  className="x-4 py-2 rounded cursor-pointer text-cyan-900 text-5xl md:text-7xl lg:text-8xl mb-8"
+                  onClick={() => {
+                    navigate("/new");
+                  }}
+                />
+              </div>
+            </div>
+            <div className="mt-12 grid sm:grid-cols-2 lg:grid-cols-3 gap-8">
+              {data.map((item, index) => (
+                <div
+                  key={index}
```

Comment:
We should never use `indexes` as key in our map function. Change it with unique key.